cmd/images-back/main.go

```diff
@@ -106,3 +116,31 @@ func (env *Env) uploadHandler() echo.HandlerFunc {
 		return c.JSON(http.StatusOK, outJSON)
 	}
 }
+
+func (env *Env) listImagesHandler() echo.HandlerFunc {
+	return func(c echo.Context) error {
+		imgs, err := models.AllImages(env.db)
+		if err != nil {
+			http.Error(c.Response(), http.StatusText(500), 500)
```

Comment:
There are no need for this line. You are returning an HTTP error in `return` statement.